app/services/hyrax/qa_select_service_decorator.rb

```diff
@@ -0,0 +1,22 @@
+# OVERRIDE: Hyrax 3.4.2 fixes issues with empty authority and no implicit conversion of string into array
+
+# frozen_string_literal: true
+
+module Hyrax
+  module QaSelectServiceDecorator
+    
+    def active?(id)
+      authority.find(id)&.fetch('active') unless authority.find(id).empty?
```

Comment:
One concern I have is in regards to multiple `authority.find(id)`; if the find is expensive we're calling it twice.

I recommend the following:

```
result = authority.find(id)
return false if result.empty?
result.fetch('active') { false }
```

The above method reduces duplicate find calls (which could be expensive).  And given that we're calling `fetch` my assumption is the data is a Hash, which could raise an error if the hash does not have the `"active"` key; hence providing the fallback value (which may or may not be the correct answer for a fallback).